code/edge/src/test/java/com/adobe/marketing/mobile/RequestBuilderTest.java

```diff
@@ -315,6 +317,74 @@ public void getPayloadWithExperienceEvents_addsKonductorConfigWithStreaming_when
 		assertEquals("\n", payloadMap.get("meta.konductorConfig.streaming.lineFeed"));
 	}
 
+	@Test
+	public void getPayloadWithExperienceEvents_setsSdkConfigMeta_whenSdkConfigPresent() throws Exception {
+		List<Event> events = getSingleEvent(getExperienceEventData("value", "5dd603781b95cc18a83d42ce"));
+		requestBuilder.addSdkConfig(new SDKConfig(new Datastream("OriginalDatastreamId")));
+		JSONObject payload = requestBuilder.getPayloadWithExperienceEvents(events);
+
+		assertNotNull(payload);
+		assertNumberOfEvents(payload, 1);
+
+		Map<String, String> payloadMap = new HashMap<>();
+		addKeys("", new ObjectMapper().readTree(payload.toString()), payloadMap);
```

Comment:
cc @timkimadobe looks like this addKeys helper for unit tests is similar to the flattening one